src/data/taskList.ts

```diff
@@ -42,6 +42,7 @@ export const businessCaseTag = (intake: SystemIntakeForm): TagEnum => {
       return 'CANNOT_START';
     case 'BIZ_CASE_DRAFT_SUBMITTED':
     case 'BIZ_CASE_FINAL_NEEDED':
+      return 'COMPLETED';
```

Comment:
This line isn't needed. It's already captured by the `return 'COMPLETED'` below.